.github/workflows/build.yaml

```diff
@@ -52,12 +49,54 @@ jobs:
           conda config --append channels pyviz/label/dev
           conda config --append channels bokeh/label/dev
           conda install -y conda-build anaconda-client build
+          echo "CONDA_PREFIX=$CONDA_PREFIX" >> $GITHUB_ENV
       - name: conda build
         run: |
           bash ./scripts/build_conda.sh
-      - name: build pyodide wheels for CDN
+      - uses: actions/upload-artifact@v3
+        if: always()
+        with:
+          name: conda_build
+          path: ${{ env.CONDA_PREFIX }}/conda-bld/noarch
+          if-no-files-found: error
+      - name: conda dev deploy
+        if: (github.event_name == 'push' && (contains(steps.vars.outputs.tag, 'a') || contains(steps.vars.outputs.tag, 'b') || contains(steps.vars.outputs.tag, 'rc')))
         run: |
-          python ./scripts/build_pyodide_wheels.py
+          VERSION="$(echo "$(ls dist/*.whl)" | cut -d- -f2)"
+          FILE="$CONDA_PREFIX/conda-bld/noarch/panel-$VERSION-py_0.tar.bz2"
+          anaconda --token ${{ secrets.CONDA_UPLOAD_TOKEN }} upload --user pyviz --label=dev $FILE
+      - name: conda main deploy
+        if: (github.event_name == 'push' && !(contains(steps.vars.outputs.tag, 'a') || contains(steps.vars.outputs.tag, 'b') || contains(steps.vars.outputs.tag, 'rc')))
+        run: |
+          VERSION="$(echo "$(ls dist/*.whl)" | cut -d- -f2)"
+          FILE="$CONDA_PREFIX/conda-bld/noarch/panel-$VERSION-py_0.tar.bz2"
+          anaconda --token ${{ secrets.CONDA_UPLOAD_TOKEN }} upload --user pyviz --label=dev --label=main $FILE
+
+  npm_build:
```

Comment:
I would like this to at least be dependent on either the conda or the pip build passing.

Reply:
Was thinking about having deploy/publish step separated for the build, which needs approval and requires all the building steps to pass, so we can ensure that everything is ready before publishing. 

Inspiration for this could be https://timheuer.com/blog/add-approval-workflow-to-github-actions/

Reply:
> we can ensure that everything is ready before publishing

I'm still unsure why we'd need that. Isn't that the purpose of the dev releases?

Reply:
Things can still go wrong. The dev and final releases are fundamentally independent; e.g., a new package can be released between the dev and final releases.

This will also give extra security if a wrong tag has been pushed, you forgot to pull before tagging locally, or you forgot to merge PRs.

Reply:
Let's bring this up in a HoloViz meeting or a more general channel. We need to ensure that the repos are structured and behave consistently.